### DISCUSSION TITLE
Help: iPhone 12 user seeing a page of code when launching Augmented Reality

### DISCUSSION THREAD

**Discussion by ashhMD** [google/model-viewer#3453] [General]
The user is seeing the below code page when they try to launch Augmented Reality from an iPhone 12. What would the reason for this be?

![image](https://user-images.githubusercontent.com/90436404/168056282-a0f4d939-fe32-4481-819b-857c5a6e017a.png)

**Comment by elalish** [comment]
See https://github.com/google/model-viewer/issues/3235

**Comment by carlosefin** [comment]
I have the same problem and haven't been able to solve it. 3d models can be viewed in AR with Safari?

**Reply by elalish** [reply]
Safari and iOS Chrome both have several issues that are getting fixes rolled out as we speak. Please post device and browser versions and what behavior you're seeing so we can be sure we're talking about the same things.
